### DISCUSSION TITLE
Wired behaviour - any help appreciated

### DISCUSSION THREAD

**Discussion by hellcry37** [mKeRix/room-assistant#529] [Q&A]
Hello, I am Cip, I'am new to this room-assistant awesome piece of software.

I prepared myself a new pi zero w with room-assistant on it made it work and all was good :) I've track 2 phones for now, one is an onelpus8pro (cip) and the other one is an lgg6 (ana).

The thing that happens to me is this:
When ana phone is home then cip phone starts behaving like in the picture attached. "Cip Phone"

[Cip Phone](https://imgur.com/a/kHYh1Ll)

[Ana Phone](https://imgur.com/a/3QqLzIx)

If anyone can explain to me why is this? What is going on?

Note: When Ana phone is not home, everything behaves normally, like is shown in the pic.

**Comment by mKeRix** [comment]
I assume you are tracking via `bluetoothClassic`? If so: you could try raising the `timeoutCycles` a bit, maybe to 3 or 4. Unfortunately I don't have a specific idea in my head as to why it would behave like this.

**Comment by hellcry37** [comment]
this happen because of many linux kernel Bluetooth problems that are present on pi environment. it is fixed now. for me at least pi zero w is terrible, is making even my network unstable. i moved to a pi3 and after fixing all Bluetooth issues all is ok
